File: e_cakeshop_mobile/lib/models/lokacija.dart
// ignore_for_file: non_constant_identifier_names

import 'package:json_annotation/json_annotation.dart';
part 'lokacija.g.dart';

@JsonSerializable()
class Lokacija {
  int? lokacijaID;
  String? naziv;
  double? Latitude;
  double? Longitude;

  Lokacija();

  factory Lokacija.fromJson(Map<String, dynamic> json) =>
      _$LokacijaFromJson(json);

  Map<String, dynamic> toJson() => _$LokacijaToJson(this);
}

File: eCakeShop.Services/Database/Lokacija.cs
namespace eCakeShop.Services.Database
{
    public class Lokacija
    {
        public int LokacijaID { get; set; }
        public string Naziv { get; set; } = null!;
        public double Latitude { get; set; }
        public double Longitude { get; set; }
    }
}
